### PR TITLE
Delete sgx backtrace

### PR DESCRIPTION
**Pull request by garbageslam**
This deletes the `sgx backtrace` crate and `sgx unwind`
- We aren't using it
- It isn't known to ever have worked in hardware mode, we need to fix something with the enclave loading address compensation
- it is likely a security problem even if it works since it reveals the contents of the stack
- The unwinding code is some pretty nasty C code that may have memory unsafety problems
- Several auditors raised questions about this, and we've been planning to remove it for a while

After this commit, `mc-sgx-panic` only has a `panic=abort` configuration

---

Note this is based on #467 but I couldn't find a way to show only the final commit in this PR

### CODE REVIEW

**Comment by garbageslam** [comment]
this needs to be rebased now because of the "Squash merge" feature

**Comment by garbageslam** [comment]
now rebased on master

File: sgx/urts/Cargo.toml [hunk 1]
```diff
@@ -5,20 +5,12 @@ authors = ["MobileCoin"]
 
 [features]
 default = [ ]
```
Comment (jcape) [inline sgx/urts/Cargo.toml:7]:
```suggestion
```

**Review by jcape — APPROVED** [review]

**Review by jcape — APPROVED** [review]

**Review by eranrund — APPROVED** [review]

**Comment by mfaulk** [comment]
Screenshot from 2020-09-24 15-32-16

Nice

**Pull request merged**